Kickstarter-iOS/Features/PledgeView/Views/PledgeViewCTAContainerView.swift

```diff
@@ -292,3 +304,55 @@ private func attributedTermsText() -> NSAttributedString? {
 
   return checkoutAttributedLink(with: string)
 }
+
+private func pledgeImmediatelyText() -> NSAttributedString? {
+  let rawText = Strings.Your_payment_method_will_be_charged()
+
+  guard let text = try? NSMutableAttributedString(
+    data: Data(rawText.utf8),
+    options: [
+      .documentType: NSAttributedString.DocumentType.html,
+      .characterEncoding: String.Encoding.utf8.rawValue
+    ],
+    documentAttributes: nil
+  ) else {
+    return nil
+  }
+
+  let attributes: String.Attributes = [
+    .font: UIFont.ksr_caption2(),
+    .foregroundColor: UIColor.ksr_support_400
+  ]
+
+  let fullRange = (text.string as NSString).range(of: text.string)
+  text.addAttributes(attributes, range: fullRange)
+
+  let boldRanges = getBoldRangesFromHtmlTags(plainString: text.string, htmlString: rawText)
+  for range in boldRanges {
+    text.addAttribute(.font, value: UIFont.ksr_caption2().bolded, range: range)
+  }
+
+  return text
+}
+
+private func getBoldRangesFromHtmlTags(plainString: String, htmlString: String) -> [NSRange] {
```

Comment:
Nit: This should live in `String+SimpleHTML` or another utility file, feels too general purpose for the pledge container. (If it doesn't already exist elsewhere).